KeenUpdate  > SKO.GridPCULimiter.

Hello , after today's KeenUpdate this warning comes up . if more logs are needed for debugging , please ask . thank you  

21:17:07.2327 [INFO]   Keen: Loaded 506 Steam Inventory item definitions
21:17:19.3003 [ERROR]  Keen: System.MissingMethodException: Method not found: 'Boolean VRage.Game.ModAPI.Interfaces.IMyDestroyableObject.DoDamage(Single, VRage.Utils.MyStringHash, Boolean, System.Nullable`1<VRage.Game.ModAPI.MyHitInfo>, Int64, Int64, Boolean)'.
   at SKO.GridPCULimiter.SKOGridPCULimiterPlugin.OnConnectorStateChanged(Int64 entityId, Int64 gridId, String entityName, String gridName, Int64 otherEntityId, Int64 otherGridId, String otherEntityName, String otherGridName, Boolean isConnected)
   at Sandbox.Game.Entities.Cube.MyShipConnector.Detach(Boolean synchronize)
   at Sandbox.Game.Entities.Cube.MyShipConnector.Detach(Boolean synchronize)
   at Sandbox.Game.Entities.Cube.MyShipConnector.TryDisconnect()
   at Sandbox.Game.GameSystems.MyGridConveyorSystem.ToggleConnectors(Nullable`1 targetState, Boolean forceToggle)
   at Sandbox.Game.Entities.MyCubeGrid.SetParkedStateInternal(Boolean v)
   at Sandbox.Game.Entities.MyCubeGrid.ToggleParkStateRequest()
   at VRage.Network.CallSite`7.Invoke(BitStream stream, Object obj, Boolean validate)
   at VRage.Network.MyReplicationLayer.Invoke(CallSite callSite, BitStream stream, Object obj, EndpointId source, MyClientStateBase clientState, Boolean validate)
   at Patched_VRage.Network.MyReplicationServerOnEvent_0(Object site, MyPacketDataBitStreamBase obj, CallSite sendAs, Object position, IMyNetObject source, Nullable`1 , EndpointId )
   at VRage.Network.MyReplicationLayer.OnEvent(MyPacketDataBitStreamBase data, NetworkId networkId, NetworkId blockedNetId, UInt32 eventId, EndpointId sender, Nullable`1 position)
   at VRage.Network.MyReplicationLayer.ProcessEvent(MyPacketDataBitStreamBase data, EndpointId sender)
   at VRage.Network.MyReplicationLayer.OnEvent(MyPacket packet)
   at Sandbox.Engine.Multiplayer.MyTransportLayer.ProcessMessage(MyPacket p)
   at Sandbox.Engine.Multiplayer.MyTransportLayer.HandleMessage(MyPacket p)
   at Sandbox.Engine.Networking.MyReceiveQueue.Process(NetworkMessageDelegate handler)
System.MissingMethodException: Method not found: 'Boolean VRage.Game.ModAPI.Interfaces.IMyDestroyableObject.DoDamage(Single, VRage.Utils.MyStringHash, Boolean, System.Nullable`1<VRage.Game.ModAPI.MyHitInfo>, Int64, Int64, Boolean)'.
   at SKO.GridPCULimiter.SKOGridPCULimiterPlugin.OnConnectorStateChanged(Int64 entityId, Int64 gridId, String entityName, String gridName, Int64 otherEntityId, Int64 otherGridId, String otherEntityName, String otherGridName, Boolean isConnected)
   at Sandbox.Game.Entities.Cube.MyShipConnector.Detach(Boolean synchronize)
   at Sandbox.Game.Entities.Cube.MyShipConnector.Detach(Boolean synchronize)
   at Sandbox.Game.Entities.Cube.MyShipConnector.TryDisconnect()
   at Sandbox.Game.GameSystems.MyGridConveyorSystem.ToggleConnectors(Nullable`1 targetState, Boolean forceToggle)
   at Sandbox.Game.Entities.MyCubeGrid.SetParkedStateInternal(Boolean v)
   at Sandbox.Game.Entities.MyCubeGrid.ToggleParkStateRequest()
   at VRage.Network.CallSite`7.Invoke(BitStream stream, Object obj, Boolean validate)
   at VRage.Network.MyReplicationLayer.Invoke(CallSite callSite, BitStream stream, Object obj, EndpointId source, MyClientStateBase clientState, Boolean validate)
   at Patched_VRage.Network.MyReplicationServerOnEvent_0(Object site, MyPacketDataBitStreamBase obj, CallSite sendAs, Object position, IMyNetObject source, Nullable`1 , EndpointId )
   at VRage.Network.MyReplicationLayer.OnEvent(MyPacketDataBitStreamBase data, NetworkId networkId, NetworkId blockedNetId, UInt32 eventId, EndpointId sender, Nullable`1 position)
   at VRage.Network.MyReplicationLayer.ProcessEvent(MyPacketDataBitStreamBase data, EndpointId sender)
   at VRage.Network.MyReplicationLayer.OnEvent(MyPacket packet)
   at Sandbox.Engine.Multiplayer.MyTransportLayer.ProcessMessage(MyPacket p)
   at Sandbox.Engine.Multiplayer.MyTransportLayer.HandleMessage(MyPacket p)
   at Sandbox.Engine.Networking.MyReceiveQueue.Process(NetworkMessageDelegate handler)

--- a/SKO-GridPCULimitter/Properties/AssemblyInfo.cs
+++ b/SKO-GridPCULimitter/Properties/AssemblyInfo.cs
@@ -31,5 +31,5 @@ using System.Runtime.InteropServices;
 // You can specify all the values or you can default the Build and Revision Numbers
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("1.5.1.0")]
-[assembly: AssemblyFileVersion("1.5.1.0")]
+[assembly: AssemblyVersion("1.5.2.0")]
+[assembly: AssemblyFileVersion("1.5.2.0")]

--- a/SKO-GridPCULimitter/manifest.xml
+++ b/SKO-GridPCULimitter/manifest.xml
@@ -4,5 +4,5 @@
 	<Name>SKO-GridPCULimiter</Name>
 	<Guid>589f0698-d9c2-4d7c-b4fb-64df67657fcd</Guid>
 	<Repository>None</Repository>
-	<Version>v1.5.1</Version>
+	<Version>v1.5.2</Version>
 </PluginManifest>

